content/en/docs/How To/create_output.md

```diff
@@ -0,0 +1,122 @@
+object and returns an instance of [Output](https://pkg.go.dev/github.com/AdRoll/baker#Output).
+
+`OutputParams` contains the index of the output and the fields `FieldIndex` that the output will
+receive during processing, in addition to the fields inherited from
+[ComponentParams](https://pkg.go.dev/github.com/AdRoll/baker#ComponentParams).
+
+`OutputParams.Index` indicates a unique index of the output process among the concurrent output
+processes generated by Baker. The `procs` configuration can be used to tune the total number of
+concurrent processes, see [Pipeline configuration](/docs/how-to/pipeline_configuration/) for details.  
+Note that the output should extensively document in `OutputDesc.Help` if it is able to manage
+concurrent processing or the user should configure it with a single process (`procs=1`).
+Read [Tuning concurrency](/docs/how-to/concurrency/) for an in-depth study on the subject.
+
+`OutputParams.Fields` is a list of [FieldIndex](https://pkg.go.dev/github.com/AdRoll/baker#FieldIndex)
+that the output will receive. Their position in this slice correspond to the fields in
+[OutputRecord.Fields](https://pkg.go.dev/github.com/AdRoll/baker#OutputRecord), see below for
+details.  
+If, for any reason, the output needs to retrieve the fields name (like the SQLite output does to
+get the columns names), then `OutputParams.FieldName` can be used.
+
+### The output configuration and help
+
+The output configuration object (`MyOutputConfig` in the previous example) must export all
+configuration parameters that the user can set in the TOML topology file.
+
+Each key in the struct must include a `help` string tag and a `required` boolean tag, both are
+mandatory.
+
+The former helps the user to understand the possible values of the field, the latter tells Baker
+whether to refuse a missing configuration param.
+
+## OutputRecord
+
+The [OutputRecord](https://pkg.go.dev/github.com/AdRoll/baker#OutputRecord) channel received by
+the output component is closed by Baker when all the records have been processed (in case of a batch
+execution) or when an interruption starts (in case of a SIGINT signal from the user).
+
+Until that moment, the output component must continuosly read new records, processing them.
+
+The `OutputRecord.Fields` slice contains the string values of the fields that the user choose to
+send to the output configuring the
+[`fields` key](http://localhost:1313/docs/how-to/pipeline_configuration/#components-configuration)
+in the `[output]` section of the TOML topology file.
+
+The fields positions correspond to the `FieldIndex` received in `OutputParams.Fields` by the
+component constructor-like function.
+
+In case of a raw output, `OutputRecord.Record` contains the full record as a byte slice.  
+The presence of raw data doesn't change the fields field value, that depend on the output
+configuration in the TOML file.
```

Comment:
Can we try put that in a single sentence for clarity?
```suggestion
In case of a raw output, `OutputRecord.Record` contains both the serialized record as a byte slice
and the field values. ```

Reply:
done